internal/options/options.go

```diff
@@ -12,68 +12,67 @@ import (
 	"github.com/launchdarkly/ld-find-code-refs/internal/version"
 )
 
-// Can't wait for contracts
 type Option interface {
 	name() string
 }
 
-type StringOption string
-type IntOption string
-type Int64Option string
-type BoolOption string
+type stringOption string
```

Comment:
made these unexported because we don't use them anywhere. this is an internal package, so this is not a breaking change